wrappers/extra_common.i

```diff
@@ -61,6 +61,9 @@
 %ignore btllib::BloomFilterInitializer;
 %ignore btllib::BloomFilterInitializer::operator=;
 
+%ignore btllib::MIBloomFilterInitializer;
+%ignore btllib::MIBloomFilterInitializer::operator=;
```

Comment:
This does fix the compile issues for the wrappers, but even though the wrappers compiled, there was an issue when I ported the miBF tests to Python in #87. Please copy the tests from there and make sure the wrappers are functional in your branch. If not, the changes in my branch will fix them. I won't merge mine into master, which is why we should combine the changes from the two branches here.

Reply:
Hello @parham-k, you are right that the miBF wrapper would throw an error during run-time after merging this PR. I fixed those issues in PR request #89. To keep PRs modular I suggest handling bugs in that PR as it is not only the miBF wrapper that suffers from bugs and they are all fixed in #89.

Reply:
Agreed, I think we should merge these two into main for the next release to avoid broken wrappers in the main branch.